baremetal/metal3machine_manager_test.go

```diff
@@ -3517,129 +3519,304 @@ var _ = Describe("Metal3Machine manager", func() {
 			expectedMD:     true,
 			expectedMDName: "md-test1",
 		}),
+		Entry("Should not find the expected MachineDeployment name, MachineSet OwnerRef Kind is not correct", testCaseGetMachineDeploymentName{
+			Machine: machineOwnedByMachineSet(),
+			MachineSetList: &capi.MachineSetList{
+				TypeMeta: metav1.TypeMeta{
+					APIVersion: capi.GroupVersion.String(),
+					Kind:       "MachineSet",
```

Comment:
Is this Kind correct? Shouldn't it be MachineSetList ?

Reply:
thanks, fixed